packages/@glimmer/core/test/non-interactive/render-test.ts

```diff
@@ -127,29 +111,44 @@ module(`[@glimmer/core] non-interactive rendering tests`, () => {
     assert.equal(await render(MainComponent), 'Glimmer!');
   });
 
-  // [true, false].forEach((pred) => {
-  //   test(`can use inline if - ${pred}`, async function(assert) {
-  //     class Main extends Component {
-  //       pred = pred;
-  //       salutation = 'Glimmer';
-  //       alternative = 'Glimmer.js';
-  //     }
+  [true, false].forEach(pred => {
+    test(`can use inline if - ${pred}`, async function(assert) {
+      class Main extends Component {
+        pred = pred;
+        salutation = 'Glimmer';
+        alternative = 'Glimmer.js';
```

Comment:
Can you make these getters, then when retrieved push into an array that we assert at the end? Specifically, I'd like the test to show that the flasey side is not called when the value is true.